### ISSUE TITLE
PyYAML related tests failure

### ISSUE THREAD

**Issue by vishal-biyani** [fission/fission#1123]
Last few PR builds and builds in master are failing around the same test case which uses PyYAML. 

```
response : a: 1
b:
  c: 3
  d: 4
2019/03/16:16:03:23 response a: 1 b: {c: 3, d: 4} not received
```

The test scripts which have the failing test case are:
- test_obj_create_in_diff_ns.sh
- test_package_command.sh
- test_buildermgr.sh 

The only plausible reason seems a new version of PyYAML (https://pypi.org/project/PyYAML/#history) and that breaks something similar to what is mentioned in [docs here](https://pyyaml.org/wiki/PyYAMLDocumentation):

```
Dictionaries without nested collections are not dumped correctly
Why does

import yaml
document = """
  a: 1
  b:
    c: 3
    d: 4
"""
print yaml.dump(yaml.load(document))
give

a: 1
b: {c: 3, d: 4}
(see #18, #24)?

```

**Comment by life1347** [comment]
PyYAML ship 5.1 5days ago and changed the default setting of yaml.load from 
```
default_flow_style=None -> default_flow_style=False
```
Here is the related PR: https://github.com/yaml/pyyaml/pull/256
I get this fixed in PR #1086

**Comment by life1347** [comment]
Fixed. Close.
